PlayCover/Model/PlayApp.swift

```diff
@@ -66,8 +66,10 @@ class PlayApp: BaseApp {
     func runAppExec() {
         let config = NSWorkspace.OpenConfiguration()
 
+        config.environment["DYLD_LIBRARY_PATH"] = "/System/iOSSupport/System/Library/Frameworks"
```

Comment:
It would be best to have this be a setting under bypasses, as the environment should not be modified in the launch process. The if-statement below was a mistake when merging one of my PRs (and will be fixed here https://github.com/PlayCover/PlayCover/pull/1130/files#diff-8c70684d8da2457c4a34d4f5f9bf6124ac596cd5cdee1fc8a04a0ba2bef9d0f7L69).

Reply:
I believe the problem is that MacCatalyst only supports linking with linked frameworks(I guess it is supported by dyld on Mac). In the iOS environment, iOS Frameworks are located in /System/Library/Frameworks. However, in MacCatalyst, that path is designated for MacOS, while iOS Frameworks exist in /System/iOSSupport/System/Library/Frameworks.
Since MacCatalyst does not support a "chroot" like functionality, it is unable to dynamically load System Plugin or Frameworks for iOS using the FileSystem. Linking via the filesystem is beneficial for certain non-objc or non-swift iOS apps, but due to the different file structure and lack of full chrooting in Mac, this may not be fully compatible. Therefore, I believe it should be the default, but I will follow your rules.

Reply:
https://github.com/Ryu-ga/PlayCover/commit/25dcbac0d83ba0ae957cad8798c812d2dffcc23b,
It can be done while install phase.